src/w3c/headers.js

```diff
@@ -721,3 +779,17 @@ function normalizeOrcid(orcid) {
 function isElement(node) {
   return node.nodeType === Node.ELEMENT_NODE;
 }
+
+function hasGitHubIssuesLink(conf) {
+  return (
+    conf.github ||
+    (conf.otherLinks &&
+      conf.otherLinks.find(linkGroup =>
+        linkGroup.data.find(
+          l =>
+            l.href &&
+            l.href.toString().match(/^https:\/\/github\.com\/.*\/issues/)
+        )
+      ))
+  );
```

Comment:
I think we should exclusively key off `conf.github`. Using other links this way should be considered deprecated.

Reply:
I think deprecating using other links for github issues sounds like a good idea, but I don't think it needs to be done at the same time as the already potentially disruptive change to process 2020.

In other words, if a spec already complies with that part of pubrules using `otherLinks` rather than `github`, I don't see any reason this should be treated as an error. Having a deprecation plan for `otherLinks` referring to github issues should be handled separately (and likely through a warning rather than an error)

Reply:
Yeah, I agree. Will do that as a followup.